### DISCUSSION TITLE
Switch between Workspaces Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#791] [Ideas]
### What feature would you like to see?

Currently we have the cycle between workspaces shortcut - but it's crucial that we have the following:

Navigate to Previous Workspace: Ctrl + Alt + Left Arrow
Navigate to Next Workspace: Ctrl + Alt + Right Arrow 

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/579_

**Comment by AntoniosBarotsis** [comment]
I'm currently using Vivaldi mostly for their workspace features. Something that works great is the ability to set keybinds for switching to specific workspaces as well, I would love to see that added in Zen.
![image](https://github.com/user-attachments/assets/09c0d2df-aab6-4c30-8cf3-3b57678819a5).

This does not sound that hard to implement since cycling through workspaces is already a thing but I could not for the life of me figure out where the code for that is 😅.

**Comment by unkindypie** [comment]
This feature is must have. I cannot switch to zen browser without this features basically, I believe there are many people like me

**Comment by vdawg-git** [comment]
A runner like in Vivaldi would be so awesome. 

So that you can press a short-cut to open the runner and then just type the first two letters of the workspace, enter and voila you switched to the workspace :D

**Comment by unkindypie** [comment]
Any updates on it?

**Reply by JSGRD22** [reply]
Added in 1.0.1-a.4.
![image](https://github.com/user-attachments/assets/1fa1846f-b9e6-4b95-ace9-0dbd0a399bad)

**Reply by marcus-sa** [reply]
@JSGRD22 does not work. Alt + Num just navigates between tabs.

**Reply by vraravam** [reply]
i can confirm that this works in macos.
